diff --git a/libfprint/drivers/elanmoc2/elanmoc2.c b/libfprint/drivers/elanmoc2/elanmoc2.c
index 62a2c1a..2b0c847 100644
--- a/libfprint/drivers/elanmoc2/elanmoc2.c
+++ b/libfprint/drivers/elanmoc2/elanmoc2.c
@@ -1455,6 +1455,7 @@ static const FpIdEntry elanmoc2_id_table[] = {
   {.vid = ELANMOC2_VEND_ID, .pid = 0x0c00, .driver_data = ELANMOC2_ALL_DEV},
   {.vid = ELANMOC2_VEND_ID, .pid = 0x0c4c, .driver_data = ELANMOC2_ALL_DEV},
   {.vid = ELANMOC2_VEND_ID, .pid = 0x0c5e, .driver_data = ELANMOC2_DEV_0C5E},
+  {.vid = ELANMOC2_VEND_ID, .pid = 0x0c6c, .driver_data = ELANMOC2_ALL_DEV},
   {.vid = 0, .pid = 0, .driver_data = 0}
 };

